docs/source/index.rst

```diff
@@ -0,0 +1,18 @@
+ESI Documentation
+=================
+
+Welcome to the documentation for Elastic Secure Infrastructure (ESI)!
+
+ESI is an OpenStack architecture that allows multiple bare metal node owners to collaborate to form a single bare metal cloud. Owners have exclusive use of their nodes, and can also lease their nodes out to lessees. Take a look at our presentation for a good high-level overview:
```

Comment:
s/architecture/project/, maybe? I'm not sure what "an architecture" is...

Reply:
I think I was faking fancy. Fixed!